.changeset/small-fishes-sing.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/design-tokens': minor
+'@twilio-paste/core': minor
+---
+
+[Design tokens] Add updated build and declaration file for gatsby.
```

Comment:
question: does this make sense to a customer? Does it reflect what we actually added to the package? These are changelog entries that we display on the website to our customers so I worry this might not make much sense to them

Reply:
✂️ @SiTaggart What do you think of the following edits?
```
[Design tokens] Add new build of design tokens formatted for compatibility with our documentation website.
- CommonJS export
- complimentary declaration file included
- Maps token buckets/categories (e.g. "background-colors") to a list of token objects.
```